src/components/05-form/search.njk

```diff
@@ -30,3 +30,23 @@
     </div>
   </form>
 </div>
+
+<h4>SDS Inverse Dropdown + Search Example</h4>
+<div class="sds-search inverse">
+  <form class="usa-search usa-search--small">
+    <label class="usa-sr-only"  for="inverse-search-options">Select Here</label>
+    <select class="usa-select inverse-search" name="options" id="inverse-search-options">
```

Comment:
if we are indicating that this is a `inverse` search at the parent container - do we need `inverse-search`?